dynawo/sources/ModelicaCompiler/Scripts_OMC_1_13_2/utils.py

```diff
@@ -856,6 +856,20 @@ def transform_atan3_operator_evalf(line):
 
     return line_tmp_bis
 
+##
+# Transform _event_floor(x, index, data) to (modelica_integer)floor(x)
```

Comment:
Please adapt the comment for readability and maintenance purposes.

Reply:
done

Reply:
Thanks